src/indent.cpp

```diff
@@ -2493,7 +2514,10 @@ void indent_text(void)
          if (chunk_is_token(prev2, CT_NEWLINE) && in_shift)
          {
             shiftcontcol = calc_indent_continue(frm);
-            frm.top().indent_cont = true;
+            // setting this in the top context when the indent is not also set just leads to compications
+            // when succeeding statements try to indent based on being embedded in a continuation
+            // or in other words it should only be set if frm.top().indent is also set
+            //frm.top().indent_cont = true;
```

Comment:
I would embed `frm.top().indent_cont` in the comment message instead to avoid confusion with dead code.